app/main.ipynb

```diff
@@ -216,14 +275,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
+   "execution_count": 5,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "array([2. , 8.4])"
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "A.mean(axis=0)"
    ],
```

Comment:
This task is for axis Y, you use the same axis and get the same result(it's incorrect). Something is wrong